### ISSUE TITLE
Unable to run scripts/release.sh in Jenkins (not a TTY)

### ISSUE THREAD

**Issue by ncdc** [containernetworking/plugins#183]
I'm trying to run `scripts/release.sh` in Jenkins, but this fails because the script has `docker run -t` and Jenkins executes its builds without a TTY. Would you accept a PR to remove the `-t`, or to only set the flag if `tty -s` returns a 0 exit code?

**Comment by rdodev** [comment]
Yup. I had to do this in other scripts (remove the `-t -i`). Go ahead.

**Comment by ncdc** [comment]
cc @squeed @dcbw

**Comment by squeed** [comment]
good catch, no need for a tty.

**Comment by ncdc** [comment]
I'll do a PR, thanks!
